src/DynamoCore/Configuration/DebugModes.cs

```diff
@@ -47,7 +47,7 @@ internal static void AddDebugMode(string name, string description, bool isEnable
         private static void RegisterDebugModes()
         {
             // Register app wide new debug modes here.
-            AddDebugMode("DynamoPreferencesMenuDebugMode", "Enable/Disable the Preferences MenuItem in the Dynamo menu.", true);
+            AddDebugMode("DynamoPreferencesMenuDebugMode", "Enable/Disable the Preferences MenuItem in the Dynamo menu.", false);
```

Comment:
Can you update the description of the debug mode to `Enable/Disable the Preferences Panel new features in the Dynamo menu.`

Reply:
Just pushed an update for this